kernel/signature.mli

```diff
@@ -84,7 +84,9 @@ type rw_infos =
     ty            : term;
     (** The type of a symbol *)
     rules         : rule_infos list;
-    (** The list of rules associated to a symbol. They are ordored by their declaration within a file and in order they are imported in the signature *)
+    (** The lifo pile of rules associated to a symbol.
```

Comment:
What is this "lifo pile" terminology here?
This data structure is called a stack in English.
And a the "fifo" version is called a queue.